.github/issue_template.md

```diff
@@ -1,7 +1,15 @@
+###
+### IMPORTANT: Please file new issues over in our Jira issue tracker!
+###
+### https://broadworkbench.atlassian.net/projects/BA/issues
+###
+### You may need to create an account before you can view/create issues.
```

Comment:
TOL: Should we add that they can use their Gmail account for the same purpose?

Reply:
Oops, I missed this before merging. Could be a good follow up though (but is it true? I thought they could create an account *using* their gmail, but not sign in with it? Maybe my confusion is exactly why we should add something!)

Reply:
You can sign in with any Google account, there is no user facing "account creation" sequence per se though obviously behind the scenes JIRA is registering you in its user database